installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.adoc

```diff
@@ -0,0 +1,33 @@
+:_content-type: ASSEMBLY
+[id="understanding-disconnected-installation-mirroring"]
+= Understanding disconnected installation mirroring
+include::_attributes/common-attributes.adoc[]
+:context: understanding-disconnected-installation-mirroring
+
+toc::[]
+// Reusing applicable content from Disconnected installation mirroring assembly
+
+You can use a mirror registry to ensure that your clusters only use container images that satisfy your organizational controls on external content. Before you install a cluster on infrastructure that you provision in a disconnected environment, you must mirror the required container images into that environment. To mirror container images, you must have a registry for mirroring.
```

Comment:
My understanding is that before external content is used, it is first vetted/approved by the organization. How do you think we should phrase this , would like to know your thoughts?

Reply:
It seems that the main purpose of the mirror registry is to remove the requirement for external access during the installation process, although yes it also can be used to limit the external content to what was mirrored locally. So maybe both purposes could be described.